app/models/crime_application.rb

```diff
@@ -5,6 +5,12 @@ class CrimeApplication < ApplicationRecord
   has_many :people, dependent: :destroy
   has_many :addresses, through: :people
 
+  scope :viewable, -> { where(status: ApplicationStatus.viewable_statuses) }
+
   enum status: ApplicationStatus.enum_values,
        _default: ApplicationStatus.enum_values[:initialised]
+
+  def pretty_status
+    status.titleize.upcase
```

Comment:
Never rely on `titleize` (or similars like `humanize`, etc.) for human presented copy or translations (in fact it is not translating), it can have unexpected side effects 😬 as it is coupling the internal identifier, that now just happens to be the same we present, but it might not be in the future. Better to always go through locales.

Reply:
Yep makes sense 👍